packages/amplication-client/src/Entity/EntityList.tsx

```diff
@@ -148,7 +150,7 @@ const EntityList: React.FC<Props> = ({ match, innerRoutes }) => {
         </div>
         {loading && <CircularProgress centerToParent />}
 
-        {!subscription && !hideNotifications.hasAccess && (
+        {isFreePlane && !hideNotifications.hasAccess && (
```

Comment:
`isFreePlane && ` seems redundant since the `hideNotifications` call already verifies the entitlement